### DISCUSSION TITLE
Correction of "Command-line interface" document.

### DISCUSSION THREAD

**Discussion by june-in-exile** [privacy-ethereum/maci#1898] [Q&A]
As I read the guide in "[Command-line interface](https://maci.pse.dev/docs/technical-references/typescript-code/cli#coordinator-set-verifying-keys-1)" document, I found that there isn’t a `maci/contracts` path in this project. The correct path should be `maci/packages/cli`.

![image](https://github.com/user-attachments/assets/c90d7413-d769-4496-ad9e-887b35ecee37)

Additionally, I tried running `pnpm run hardhat` in `maci/packages/cli`, but it failed (as shown in the screenshot). I guess the proper instruction here is `npx hardhat node`?

<img width="440" alt="image" src="https://github.com/user-attachments/assets/dba08fd8-9a2b-4c36-abd7-b314cf80d50b">

**Comment by ctrlc03** [comment]
> As I read the guide in "[Command-line interface](https://maci.pse.dev/docs/technical-references/typescript-code/cli#coordinator-set-verifying-keys-1)" document, I found that there isn’t a `maci/contracts` path in this project. The correct path should be `maci/packages/cli`.
> 
> ![image](https://private-user-images.githubusercontent.com/26154393/383335944-c90d7413-d769-4496-ad9e-887b35ecee37.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzMjUxMzMsIm5iZiI6MTczMzMyNDgzMywicGF0aCI6Ii8yNjE1NDM5My8zODMzMzU5NDQtYzkwZDc0MTMtZDc2OS00NDk2LWFkOWUtODg3YjM1ZWNlZTM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA0VDE1MDcxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1NzlkNTJjODNiNDJkNjU2MDBjOWY0ZmE5OGQ0M2VmMjY5MDEwMmNlMzA3OGM5NzI1NzkyMjU1ZjQ2MGFhOWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.d4QTK-ACMGmBiSpY1k74cM8oYIvX1JagTy9Uaf8I08c)
> 
> Additionally, I tried running `pnpm run hardhat` in `maci/packages/cli`, but it failed (as shown in the screenshot). I guess the proper instruction here is `npx hardhat node`?
> 
> <img alt="image" width="440" src="https://private-user-images.githubusercontent.com/26154393/383334710-dba08fd8-9a2b-4c36-abd7-b314cf80d50b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzMjUxMzMsIm5iZiI6MTczMzMyNDgzMywicGF0aCI6Ii8yNjE1NDM5My8zODMzMzQ3MTAtZGJhMDhmZDgtOWEyYi00YzM2LWFiZDctYjMxNGNmODBkNTBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA0VDE1MDcxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxMTcyZDUyOGY3NDNmMWEzNThiZDJkZjUwM2M1YWY2YzRjZDNhY2ZmNzBjN2JlMjRjNDFhMDlhYmJjOGZhOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.56kFTGc8BslxDqixAISJkkklXUWvLvfQ17hkm5LNYkw">

Hi @june-in-exile - next time I'd recommend you use the issues section of the repo https://github.com/privacy-scaling-explorations/maci/issues for such errors. So actually the correct path should be packages/contracts not cli - does that help?
